worblehat-domain/src/test/java/de/codecentric/psd/worblehat/domain/StandardBookServiceTest.java

```diff
@@ -28,18 +27,29 @@
 
 	private static final DateTime NOW = DateTime.now();
 
-	private static final Book TEST_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book A_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book ANOTHER_BOOK = new Book("title2", "author2", "edition2", "isbn2", 2016);
+
+	private static final Book A_BORROWED_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book ANOTHER_BORROWED_BOOK = new Book("title2", "author2", "edition2", "isbn2", 2016);
+	Set<Book> books;
+
+	{
+		A_BORROWED_BOOK.borrowNowByBorrower(BORROWER_EMAIL);
+		ANOTHER_BORROWED_BOOK.borrowNowByBorrower(BORROWER_EMAIL);
+	}
```

Comment:
Never saw a block used like this. I didn't even know this works! I always thought you need to use a `static` block for this.

Reply:
I thought this is a static block? O:-) Looks like I accidentally created an instance initializer: https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.6